src/main/java/mate/academy/springboot/swagger/model/Product.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springboot.swagger.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode

```

Comment:
In this case you better write @Date instead of getter, setter, toString, equalsandhashcode

Reply:
Using @Data for JPA entities is not recommended. It can cause severe performance and memory consumption issues.

Reply:
I know but you are using toString and EqualsAndHashCode annotations. It's also bad practice for enteties.
Data =(setter, getter, equalsandhashcode, toString). Because of that I wrote you.
So my recommendations: Leave only Setter and Getter.